find.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
What is this script for / where is this used? Can we move that into the `supporting_scripts` folder?

Reply:
shit